dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcAnalyticsTableManager.java

```diff
@@ -369,6 +371,23 @@ private void populateTable(
             + "inner join _categorystructure acs on dv.attributeoptioncomboid=acs.categoryoptioncomboid "
             + "inner join _categoryoptioncomboname aon on dv.attributeoptioncomboid=aon.categoryoptioncomboid "
             + "inner join _categoryoptioncomboname con on dv.categoryoptioncomboid=con.categoryoptioncomboid "
+            + "left join (select dv.dataelementid as dataelementid,"
+            + "                  dv.sourceid as sourceid,"
+            + "                  dv.categoryoptioncomboid as categoryoptioncomboid,"
+            + "                  dv.attributeoptioncomboid as attributeoptioncomboid,"
+            + "                  avg(dv.value::double precision) as avg_middle_value,"
+            + "                  percentile_cont(0.5)"
+            + "                            within group (order by dv.value::double precision) as percentile_middle_value,"
+            + "                            stddev_pop(dv.value::double precision) as std_dev"
+            + "           from datavalue dv"
```

Comment:
We could rename the `inner` "dv" alias, to like `dv1`, or something. Just to make it easier to read.

Reply:
accept

Reply:
done